website/server/models/task.js

```diff
@@ -236,6 +236,8 @@ export let DailySchema = new Schema(_.defaults({
     su: {type: Boolean, default: true},
   },
   streak: {type: Number, default: 0},
+  daysOfMonth: {type: [Number], default: []}, // Days of the month that the daily should repeat on
```

Comment:
What if invalid values are given? Ex: day 10 of the week, day 40 of the month, etc.  Or even day 30 might be problematic during February.